src/components/ui/MonacoEditor.svelte

```diff
@@ -55,7 +55,7 @@
   let div: HTMLDivElement;
   let editor: Editor.IStandaloneCodeEditor | undefined = undefined;
   let codeActionProvider: IDisposable | undefined = undefined;
-  let editor_load_event: IDisposable | undefined = undefined;
+  let editor_load_event: IDisposable | undefined = undefined; // TODO this does not appear to be used.
```

Comment:
Yeah I'm ok with deleting it. It doesn't look like it was even used when it was first added.